src/Services/DataService.test.ts

```diff
@@ -690,7 +757,52 @@ describe("DataService", () => {
       fakeService.updateRecordEpic(ActionsObservable.of(updateRecordAction), store)
         .subscribe(noop, noop,
           () => {
-            assert.isTrue(mockAdapter.updateItem.calledWithMatch(expectedResult.id, JSON.stringify({ fullText: "puppy" })));
+            expect(mockAdapter.updateItem.calledWithMatch(expectedResult.id, JSON.stringify({ fullText: "puppy" }))).to.be.true;
+          });
+    });
+
+    it("updateRecordEpic should call normailize after deserialize", () => {
+      const onSuccess = spy();
+      const expectedResult = { id: "123", fullText: "zella puppy" };
+      const updateRecordAction = fakeService.actions.updateRecord(expectedResult, { onSuccess });
+      const normalizedStub = stub(fakeService.mapper, "normalize");
+
+      stub(fakeService.serializer, "deserialize").returns(expectedResult);
+
+      fakeService.updateRecordEpic(ActionsObservable.of(updateRecordAction), store)
+        .subscribe(noop, noop,
+          () => {
+            expect(normalizedStub.firstCall.args[0]).to.equal(expectedResult);
+          });
+    });
+
+    it("updateRecordEpic should call transform before serialize", () => {
+      const onSuccess = spy();
+      const expectedResult = { id: "123", fullText: "zella puppy transform" };
+      const updateRecordAction = fakeService.actions.updateRecord(expectedResult, { onSuccess });
+      const transformStub = stub(fakeService.mapper, "transform");
+
+      stub(fakeService.serializer, "serialize").returns(expectedResult);
```

Comment:
This is the opposite of what we want.  We want to make sure `serialize` is called with the result of `transform`

Reply:
I think that test is down below this one. On line 794.

Reply:
Looks like I have 2 tests doing the same thing. Maybe one needs to be deleted? Not sure.